kimchi/Cargo.toml

```diff
@@ -13,6 +13,10 @@ license = "Apache-2.0"
 path = "src/lib.rs"
 bench = false       # needed for criterion (https://bheisler.github.io/criterion.rs/book/faq.html#cargo-bench-gives-unrecognized-option-errors-for-valid-command-line-options)
 
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
```

Comment:
Nit/unrelated to this patch: maybe we should also use this instead of a feature `wasm-types` for the rest of the WebAssembly code. Just a thought, not sure if it would work.